rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -196,6 +196,20 @@ void ObjectRenderer::renderInstance(InstanceObject *instance,
 		return;
 	}
 
+
+	if (instance->object->timeOn != 0 || instance->object->timeOff != 0)
```

Comment:
We set both timeOn and timeOff to zero for normal OBJs, so without this line, none of the normal OBJs are rendered.